src/utils/isEmail.js

```diff
@@ -0,0 +1,15 @@
+import isEmpty from './isEmpty';
+import LesgoException from '../exceptions/LesgoException';
+
+export default email => {
+  const pattern = /^[\w\-\.\+]+\@[a-zA-Z0-9\.\-]+\.[a-zA-z0-9]{2,5}$/; // eslint-disable-line no-useless-escape
+
+  if (isEmpty(email)) {
```

Comment:
Since you're using typescript might be better to be explicit as to what is the type of email argument here. By doing that you don't need to use isEmpty

Reply:
Nope, we are not using typescript. TypeScript was used on a different project, not the lesgo-framework. Eventually we'll get there